### PR TITLE
[INTERNAL] Use JSON.parse to fetch version number

### PR DESCRIPTION
**Pull request by locks**
The json gem has been part of the Ruby stdlib for a while now.
With this change, we mirror what is done in the Ember Data codebase.

### CODE REVIEW

File: ember-source.gemspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/emberjs/ember.js"
   gem.license       = 'MIT'
 
-  gem.version       = Ember.rubygems_version_string
+  gem.version       = Ember::VERSION
 
   gem.files = %w(VERSION) + Dir['dist/*.js', 'lib/ember/*.rb']
```
Comment (rwjblue) [inline ember-source.gemspec]: I think this also needs updating?

Comment (locks) [reply ember-source.gemspec]: Added `package.json`.
Question, though, do we still need that `VERSION` file?

Comment (locks) [reply ember-source.gemspec]: It seems to be in use in the deploy script:

    gem push ember-source-`cat VERSION`.gem

**Comment by Serabe** [comment]
Restarted sauce :(

**Comment by locks** [comment]
It's green, can someone review?

**Comment by stefanpenner** [comment]
:+1:

**Pull request merged**